sdk/maps/azure-maps-render/README.md

```diff
@@ -0,0 +1,158 @@
+# Azure Resource Manager Render client library for Java
+
+Azure Resource Manager Render client library for Java.
+
+This package contains Microsoft Azure SDK for Render Management SDK. Azure Maps Render REST APIs. Package tag 2.1-preview. For documentation on how to use this package, please see [Azure Management Libraries for Java](https://aka.ms/azsdk/java/mgmt).
+
+## We'd love to hear your feedback
+
+We're always working on improving our products and the way we communicate with our users. So we'd love to learn what's working and how we can do better.
+
+If you haven't already, please take a few minutes to [complete this short survey][survey] we have put together.
+
+Thank you in advance for your collaboration. We really appreciate your time!
+
+## Documentation
+
+Various documentation is available to help you get started
+
+- [API reference documentation][docs]
```

Comment:
Suggest to update the doc link to [Azure Maps Search API](https://docs.microsoft.com/rest/api/maps/search) page.